### ISSUE TITLE
Not working in android 6.0 (sony update)

### ISSUE THREAD

**Issue by NguyenVanBaoHaui** [daimajia/AndroidSwipeLayout#385]
after swipe it turn back to before change .

**Comment by HusseinElFeky** [comment]
Yes, I have the same issue too. I tested it on Xperia XA Ultra 6.0. When I swipe the view, the back view gets hidden instantly although its touching part is working. It's like the drawing of the view is just not rendered. It's a very weird bug. I hope someone can fix it.

**Comment by pmSven** [comment]
i fixed the problem by adding this line into the onBindViewHolder Method of my RecyclerSwipeAdapter:

mItemManger.bindView(holder.itemView, position); //needed to work on sony devices with android6

**Comment by vicensq** [comment]
I have the same issue with Cubot Android 6.0. I used ArraySwipeAdapter, not RecyclerSwipeAdapter.
How can fix this problem?

Thanks

**Comment by appukrb** [comment]
@pmSven In this mItemManger.bindView(holder.itemView, position);  What is mItemManger?

**Comment by pmSven** [comment]
its a member of the RecyclerSwipeAdapter.
`public abstract class RecyclerSwipeAdapter<VH extends ViewHolder> extends Adapter<VH> implements SwipeItemMangerInterface, SwipeAdapterInterface {
    public SwipeItemRecyclerMangerImpl mItemManger = new SwipeItemRecyclerMangerImpl(this);
`
